### PR TITLE
Plot rework for the verifier-opt-plot

### PR DESCRIPTION
**Pull request by cormensratio**
Adapts the `b_verifier_report_opt_plot` plot to the click CLI.

### CODE REVIEW

**Comment by cormensratio** [comment]
Currently untested due to an error in the generation of BlameVerifierOpt reports:
...benchbuild/environments/entrypoints/cli.py", line 15, in <module> rich.traceback.install() AttributeError: module 'rich' has no attribute 'traceback'

**Review by vulder — APPROVED** [review]
lgtm

File: varats/varats/plots/blame_verifier_report_plots.py [hunk 1]
```diff
@@ -10,23 +10,27 @@
...
+)
 from varats.plots.case_study_overview import SUCCESS_COLOR, FAILED_COLOR
 from varats.utils.git_util import FullCommitHash
 
 LOG = logging.getLogger(__name__)
 
 
 def _get_named_df_for_case_study(
-    case_study: CaseStudy, opt_level: OptLevel
+    case_study: CaseStudy, opt_level: OptLevel, plot_kwargs: tp.Any
```
Comment (boehmseb) [inline varats/varats/plots/blame_verifier_report_plots.py]: Same kwargs typing issue as mentioned in one of the other PRs.

**Review by boehmseb — CHANGES_REQUESTED** [review]

File: varats/varats/plots/blame_verifier_report_plots.py (hunk 1 above)
Comment (cormensratio) [reply varats/varats/plots/blame_verifier_report_plots.py]: Done

File: varats/varats/plots/blame_verifier_report_plots.py [hunk 2]
```diff
@@ -43,9 +48,10 @@ def _get_named_df_for_case_study(
     verifier_plot_df = verifier_plot_df.loc[verifier_plot_df['opt_level'] ==
                                             opt_level.value]
     if verifier_plot_df.empty or len(
-        np.unique(verifier_plot_df['revision'])
+        np.unique([rev.hash for rev in verifier_plot_df['revision']]
+                 )  # type: ignore
```
Comment (boehmseb) [inline varats/varats/plots/blame_verifier_report_plots.py]:
```suggestion
        verifier_plot_df['revision'].unique()
```

**Review by boehmseb — APPROVED** [review]
After you fix the small suggestion we can merge this

File: varats/varats/plots/blame_verifier_report_plots.py (hunk 2 above)
Comment (cormensratio) [reply varats/varats/plots/blame_verifier_report_plots.py]: That's better ;)

**Pull request merged**